### PR TITLE
Initial commit of Hyperledger Indy Chapter and Demo Code

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

Initial commit of the Hyperledger Indy chapter, including a still early draft of the chapter text, and an in progress code example using Indy-Node and Indy-SDK as the basis of some web-based Agents.

More to do - edits and cleanup, add images, add questions, add references and continue to add guidance around the demo.

### CODE REVIEW

File: LFS171x/docs/introduction-to-hyperledger-indy.md [hunk 1]
```diff
@@ -0,0 +1,689 @@
...
+
+Hyperledger Indy has been created to add an Identity layer to the Internet using a mechanism that is easy to use, enables online trust, and enhances privacy. It's a big goal that is of vital importance to everyone on the Internet. And, it's a goal that has recently become realizable with the advent of blockchain.
+
+In this chapter, we'll look in more detail at the identity problem and how Hyperledger Indy uses Decentralized Identifiers (DIDs) and Verifiable Credentials to add Identity as a core component of the Internet. We'll look at the blockchain elements of Hyperledger Indy, comparing Indy's approach to blockchain with the other systems we've looked at in this course.
+
+With that background, we'll go through a couple of Indy demonstrations - a video showing DIDs and Verifiable Credentials being used, and running a local instance of Indy that demonstrates Indy's software Agents communicating to prove who's who on the Internet. In doing that, we'll dig into to a question that is important to everyone that looks into Indy (and other Blockchains) - what goes on the ledger?
```
Comment (tkuhrt) [inline LFS171x/docs/introduction-to-hyperledger-indy.md]: "into to" -> "into"

**Comment by thaddeus-cambron** [comment]
@tkuhrt I am having trouble with my PR

File: LFS171x/docs/introduction-to-hyperledger-indy.md (hunk 1 above)
Comment (swcurran) [reply LFS171x/docs/introduction-to-hyperledger-indy.md]: @tkuhrt - suggest you review the Google doc for typos, content, concepts etc. as that is still in the source of the materials.  This should be used to make sure that the markdown is properly structured and to look at the images.

File: LFS171x/docs/introduction-to-hyperledger-indy.md [hunk 2]
```diff
@@ -0,0 +1,689 @@
...
+
+
+# Introduction and Learning Objectives
+
+
+## Introduction to Hyperledger Indy
+
+Welcome to the Hyperledger Indy section of Blockchain for Business!  In this chapter, you will learn about Hyperledger Indy - a blockchain-based system that is quite different from the other Hyperledger projects we've looked at so far. Where the other projects are general purpose blockchain systems (they can be used in many situations), Hyperledger Indy is used for just one purpose - and it's a big one! Indy is all about Identity on the Internet. It's about being able to prove to others who you are and you being certain who they are.
+
+![HyperledgerIndyLogo](../images/introduction-to-hyperledger-indy/HLindy.png "Hyperledger Indy Logo")
```
[the remark this answers is not in the document]
Comment (swcurran) [reply LFS171x/docs/introduction-to-hyperledger-indy.md]: Weird - the image file is there and correctly referenced.  Not sure what that is...  It's a png file, but there are other png images in the doc that are fine.

I'll keep looking.

BTW - how do I control the size of the images?  Not done in github-flavoured MD I don't think.  How does it work in EdX?

File: LFS171x/docs/introduction-to-hyperledger-indy.md (hunk 1 above)
Comment (thaddeus-cambron) [reply LFS171x/docs/introduction-to-hyperledger-indy.md]: I am having trouble with my PRs as well

File: LFS171x/docs/introduction-to-hyperledger-indy.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (nathalie-ckc) [reply LFS171x/docs/introduction-to-hyperledger-indy.md]: @swcurran , i was able to control size of images like this:
RocketChat Logo

**Pull request closed without merging**